src/components/PhoneNumberWithFlag.tsx

```diff
@@ -1,11 +1,13 @@
 import { getCountryEmoji, parsePhoneNumber } from '@celo/phone-utils'
 import * as React from 'react'
 import { StyleSheet, Text, View } from 'react-native'
-import fontStyles from 'src/styles/fonts'
+import Colors from 'src/styles/colors'
+import fontStyles, { typeScale } from 'src/styles/fonts'
 
 interface Props {
   e164PhoneNumber: string
   defaultCountryCode?: string
+  textColor?: string
```

Comment:
nit: enforce strict color types where possible
```suggestion
  textColor?: Colors
```